pyqtgraph/meta.yaml

```diff
@@ -1,29 +1,35 @@
+{% set name = 'pyqtgraph' %}
+{% set version = '0.9.11' %}
+{% set number = '1' %}
+
 about:
     home: http://www.pyqtgraph.org/
     license: MIT
-    summary: A pure-Python graphics library for PyQt/PySide (git master build)
+    summary: |
+        (Supervised master build)
```

Comment:
I know we talked about it, but in case anyone doesn't know, the pipe implies a long or multi-line string.